tests/getMovieById.test.ts

```diff
@@ -0,0 +1,18 @@
+import { MovieGateway } from "../src/business/gateways/movieGateways";
+import { GetMovieByIdUC } from "../src/business/UseCases/GetMovieById";
+
+const moviesGatewayImplementation: MovieGateway = {
+    getMovieById: jest.fn()
+};
+
+describe('Testando filmes', () => {
+    test("Deve retornar um erro", async () => {
```

Comment:
Quando for escrever testes, atente-se a:
- Indicar qual erro vai ser retornado ou a situação do erro ("quando um id não for enviado")
- Garantir que o teste vai fazer o que a mensagem está dizendo (no caso de vocês, não estão testando erro nenhum!)